scripts/qesap/lib/cmds.py

```diff
@@ -371,9 +371,15 @@ def cmd_ansible(configure_data, base_project, dryrun, verbose, destroy=False, pr
         if dryrun:
             print(' '.join(command['cmd']))
         else:
-            ret, _ = lib.process_manager.subprocess_run(**command)
+            ret, out = lib.process_manager.subprocess_run(**command)
             log.debug("Ansible process return ret:%d", ret)
+            playbook_path = command['cmd'][4]
+            playbook_name = os.path.splitext(os.path.basename(playbook_path))[0]
+            log_filename = f"ansible.{playbook_name}.log.txt"
+            log.debug("Write %s getcwd:%s", log_filename, os.getcwd())
+            with open(log_filename, 'w', encoding='utf-8') as log_file:
+                log_file.write('\n'.join(out))
             if ret != 0:
                 log.error("command:%s returned non zero %d", command, ret)
-                return Status(ret)
+                return Status(f"Error at {command}")
```

Comment:
why this is needed? Is it right?

Reply:
Yes it is right, just to show the command and the error

Reply:
But it is not showing the value of `ret` anymore